--- netdude-0.5.0/plugins/ChecksumFix/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/plugins/ChecksumFix/Makefile.am	2009-09-14 15:12:54.000000000 +0200
@@ -7,7 +7,7 @@
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/plugins
+pkgdir		     = $(pkglibdir)/$(version_major)/plugins
 
 pkg_LTLIBRARIES	     = nd_cksumfix.la
 
--- netdude-0.5.0/plugins/BPF-Filter/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/plugins/BPF-Filter/Makefile.am	2009-09-14 15:13:35.000000000 +0200
@@ -7,7 +7,7 @@
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/plugins
+pkgdir		     = $(pkglibdir)/$(version_major)/plugins
 
 pkg_LTLIBRARIES	     = nd_bpf.la
 
--- netdude-0.5.0/protocols/tcp/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/tcp/Makefile.am	2009-09-14 15:14:20.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/tcp
 
 pkg_LTLIBRARIES	     = nd_tcp.la 
--- netdude-0.5.0/protocols/linux-sll/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/linux-sll/Makefile.am	2009-09-14 15:14:57.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/linux-sll
 
 pkg_LTLIBRARIES	     = nd_sll.la 
--- netdude-0.5.0/protocols/fddi/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/fddi/Makefile.am	2009-09-14 15:15:33.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/fddi
 
 pkg_LTLIBRARIES	     = nd_fddi.la 
--- netdude-0.5.0/protocols/ip/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/ip/Makefile.am	2009-09-14 15:15:58.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/ip
 
 pkg_LTLIBRARIES	     = nd_ip.la 
--- netdude-0.5.0/protocols/vlan/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/vlan/Makefile.am	2009-09-14 15:16:25.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/vlan
 
 pkg_LTLIBRARIES	     = nd_vlan.la 
--- netdude-0.5.0/protocols/icmp/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/icmp/Makefile.am	2009-09-14 15:17:05.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/icmp
 
 pkg_LTLIBRARIES	     = nd_icmp.la 
--- netdude-0.5.0/protocols/ether/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/ether/Makefile.am	2009-09-14 15:17:27.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/ether
 
 pkg_LTLIBRARIES	     = nd_ethernet.la 
--- netdude-0.5.0/protocols/pcap/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/pcap/Makefile.am	2009-09-14 15:17:48.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/pcap
 
 pkg_LTLIBRARIES	     = nd_pcap.la 
--- netdude-0.5.0/protocols/arp/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/arp/Makefile.am	2009-09-14 15:18:18.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/arp
 
 pkg_LTLIBRARIES	     = nd_arp.la 
--- netdude-0.5.0/protocols/snap/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/snap/Makefile.am	2009-09-14 15:18:44.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/snap
 
 pkg_LTLIBRARIES	     = nd_snap.la 
--- netdude-0.5.0/protocols/udp/Makefile.am~	2009-09-14 15:10:25.000000000 +0200
+++ netdude-0.5.0/protocols/udp/Makefile.am	2009-09-14 15:19:15.000000000 +0200
@@ -7,7 +7,7 @@
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
                        -I$(top_srcdir)/src -W -Wall
 
-pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
+pkgdir		     = $(pkglibdir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/udp
 
 pkg_LTLIBRARIES	     = nd_udp.la 
--- netdude-0.5.0/configure.in~	2008-08-28 15:58:22.000000000 +0200
+++ netdude-0.5.0/configure.in	2009-09-14 15:22:48.000000000 +0200
@@ -80,14 +80,14 @@
 dnl ##################################################
 dnl # Set PACKAGE_DATA_DIR in config.h.
 dnl ##################################################
-if test "x${datadir}" = 'x${prefix}/share'; then
+if test "x${libdir}" = 'x${prefix}/lib'; then
   if test "x${prefix}" = "xNONE"; then
-    packagedatadir="${ac_default_prefix}/share/${PACKAGE}"
+    packagedatadir="${ac_default_prefix}/lib/${PACKAGE}"
   else
-    packagedatadir="${prefix}/share/${PACKAGE}"
+    packagedatadir="${prefix}/lib/${PACKAGE}"
   fi
 else
-  packagedatadir="${datadir}/${PACKAGE}"
+  packagedatadir="${libdir}/${PACKAGE}"
 fi
 packagedatadir=`eval eval eval eval "echo $packagedatadir"`
 AC_DEFINE_UNQUOTED(PACKAGE_DATA_DIR, "${packagedatadir}", [The data dir of the package])
