--- netdude-0.5.0/Makefile.am~	2005-05-03 23:53:10.000000000 +0200
+++ netdude-0.5.0/Makefile.am	2009-09-14 14:55:44.000000000 +0200
@@ -1,4 +1,4 @@
-SUBDIRS = libltdl src plugins protocols pixmaps doc
+SUBDIRS = src plugins protocols pixmaps doc
 
 MAINTAINERCLEANFILES = Makefile.in aclocal.m4 config.guess \
                        config.h.in config.sub configure install-sh \
--- netdude-0.5.0/plugins/ChecksumFix/Makefile.am~	2005-03-17 15:09:45.000000000 +0100
+++ netdude-0.5.0/plugins/ChecksumFix/Makefile.am	2009-09-14 14:59:37.000000000 +0200
@@ -5,7 +5,7 @@
 
 LIBS                 =
 AM_LDFLAGS	     = -L$(top_srcdir)/src
-INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/plugins
 
--- netdude-0.5.0/plugins/BPF-Filter/Makefile.am~	2005-03-17 15:09:44.000000000 +0100
+++ netdude-0.5.0/plugins/BPF-Filter/Makefile.am	2009-09-14 15:00:32.000000000 +0200
@@ -5,7 +5,7 @@
 
 LIBS                 =
 AM_LDFLAGS	     = -L$(top_srcdir)/src
-INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/plugins
 
--- netdude-0.5.0/protocols/tcp/Makefile.am~	2005-03-17 15:09:46.000000000 +0100
+++ netdude-0.5.0/protocols/tcp/Makefile.am	2009-09-14 15:01:13.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/tcp
--- netdude-0.5.0/protocols/linux-sll/Makefile.am~	2005-03-17 15:09:46.000000000 +0100
+++ netdude-0.5.0/protocols/linux-sll/Makefile.am	2009-09-14 15:01:53.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/linux-sll
--- netdude-0.5.0/protocols/fddi/Makefile.am~	2005-03-17 15:09:45.000000000 +0100
+++ netdude-0.5.0/protocols/fddi/Makefile.am	2009-09-14 15:02:35.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/fddi
--- netdude-0.5.0/protocols/ip/Makefile.am~	2005-03-17 15:09:46.000000000 +0100
+++ netdude-0.5.0/protocols/ip/Makefile.am	2009-09-14 15:03:07.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/ip
--- netdude-0.5.0/protocols/vlan/Makefile.am~	2007-11-13 20:02:04.000000000 +0100
+++ netdude-0.5.0/protocols/vlan/Makefile.am	2009-09-14 15:03:43.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/vlan
--- netdude-0.5.0/protocols/icmp/Makefile.am~	2005-03-17 15:09:45.000000000 +0100
+++ netdude-0.5.0/protocols/icmp/Makefile.am	2009-09-14 15:04:25.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/icmp
--- netdude-0.5.0/protocols/ether/Makefile.am~	2005-03-17 15:09:45.000000000 +0100
+++ netdude-0.5.0/protocols/ether/Makefile.am	2009-09-14 15:04:55.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/ether
--- netdude-0.5.0/protocols/pcap/Makefile.am~	2007-03-06 23:43:26.000000000 +0100
+++ netdude-0.5.0/protocols/pcap/Makefile.am	2009-09-14 15:05:31.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/pcap
--- netdude-0.5.0/protocols/arp/Makefile.am~	2005-03-17 15:09:45.000000000 +0100
+++ netdude-0.5.0/protocols/arp/Makefile.am	2009-09-14 15:05:58.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/arp
--- netdude-0.5.0/protocols/snap/Makefile.am~	2005-03-17 15:09:46.000000000 +0100
+++ netdude-0.5.0/protocols/snap/Makefile.am	2009-09-14 15:06:37.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/snap
--- netdude-0.5.0/protocols/udp/Makefile.am~	2005-03-17 15:09:46.000000000 +0100
+++ netdude-0.5.0/protocols/udp/Makefile.am	2009-09-14 15:07:13.000000000 +0200
@@ -5,7 +5,7 @@
 
 AM_LDFLAGS	     = -L$(top_srcdir)/src
 INCLUDES	     = @gtk_cflags@ @LIBNETDUDE_CFLAGS@ -I$(top_srcdir) \
-                       -I$(top_srcdir)/libltdl -I$(top_srcdir)/src -W -Wall
+                       -I$(top_srcdir)/src -W -Wall
 
 pkgdir		     = $(pkgdatadir)/$(version_major)/protocols
 pkgincludedir        = $(includedir)/netdude/$(version_major)/protocols/udp
--- netdude-0.5.0/src/Makefile.am~	2008-08-28 15:54:39.000000000 +0200
+++ netdude-0.5.0/src/Makefile.am	2009-09-14 15:09:12.000000000 +0200
@@ -1,7 +1,7 @@
 ## Process this file with automake to produce Makefile.in
 
 INCLUDES = \
-	-I$(top_srcdir) -I$(top_srcdir)/libltdl \
+	-I$(top_srcdir) \
 	-I$(top_srcdir)/src @gtk_cflags@ \
 	-DLIBNETDUDE_CFLAGS="\"@LIBNETDUDE_CFLAGS@\"" \
 	-DGTK_CFLAGS="\"@gtk_cflags@\"" \
@@ -75,7 +75,7 @@
 	gtkhex.c gtkhex.h
 
 netdude_LDADD = \
-	$(top_builddir)/libltdl/libltdlc.la @gtk_libs@ @LIBNETDUDE_LIBS@
+	-lltdl @gtk_libs@ @LIBNETDUDE_LIBS@
 
 netdude_LDFLAGS = -export-dynamic
 
